Zend/zend_API.h

```diff
@@ -2015,7 +2015,8 @@ static zend_always_inline int zend_parse_arg_str_or_obj(
 ) {
 	if (EXPECTED(Z_TYPE_P(arg) == IS_OBJECT)) {
 		if (base_ce && UNEXPECTED(!instanceof_function(Z_OBJCE_P(arg), base_ce))) {
-			return 0;
+			*destination_object = NULL;
+			return zend_parse_arg_str_slow(arg, destination_string);
```

Comment:
I've split off this fix into https://github.com/php/php-src/commit/993be4b9bf34b2d1a7c288599b0e011aa9f33906.

Reply:
OK, thank you!